### DISCUSSION TITLE
gulp-stylelint dead? Any successors?

### DISCUSSION THREAD

**Discussion by markleppke** [gulpjs/gulp#2696] [Help] [answered]
I'm working on a team that used gulp for processing it's SCSS. Gulp-stylelint isn't compatible with the latest stylelint, and additionally it fails to ignore code when fix: true. Instead it replaces all of those styles with the javascript object of that file with the settings passed into gulp-stylelint. 

Needless to say its causing all sorts of issues for us and we had to disable linting. Starting to go down the path of using postCSS and postCss-reporter. it lints but doesn't error out when lint errors are present.

Other options here?

**Comment by seif3r** [comment, marked as answer]
This'll help you get it rolling again. community fork that is mostly updated.
https://github.com/ronilaukkarinen/gulp-stylelint
